Berksfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
This being a separate commit has made it all worth it. 😛 

![gcllqnv7wzklu](https://cloud.githubusercontent.com/assets/5170/26082197/8eafe84c-3983-11e7-9e88-438e872b7a2b.gif)